icssc-projects/AntAlmanac
# Braedon

**Summary**
The state `addedCourse` was created for SectionTableBody. It becomes true when the add button is clicked in a class search, or upon creation for the row, if the course is already in the list of added ones.

The html class `addedCourse` was also created, and is added to the row if the state `addedCourse` is true.

The function to set the state was also added to the objects in the addedCourses array to allow for removing highlights.

![image](https://user-images.githubusercontent.com/25188285/101305145-d8941d00-37f6-11eb-9ec6-f41c01f148d7.png)
![image](https://user-images.githubusercontent.com/25188285/101305173-e8136600-37f6-11eb-804b-09cf76c52b1d.png)


**Test Plan**
- Verify that clicking add highlights the corresponding row
- Verify that deleting a course from the schedule removes the highlight
- Verify that all courses are highlighted in the Added Classes tab
- Verify that switching schedule numbers highlights only the corresponding courses
- Verify that doing a new search retains highlights

**Issues**
#26
